cfme/storage/volume.py

```diff
@@ -105,6 +105,7 @@ class VolumeAddForm(View):
     volume_name = TextInput(name='name')
     size = TextInput(name='size')
     tenant = BootstrapSelect(name='cloud_tenant_id')
+    type = BootstrapSelect(name='volume_type')
```

Comment:
I would recommend using `volume_type` for this field to not shadow the builtin `type`

Reply:
Done